pom.xml

```diff
@@ -41,7 +41,7 @@
   <description>
     This plugin provides a new enhanced API for interacting with SCM systems.
   </description>
-  <url>https://wiki.jenkins.io/display/JENKINS/SCM+API+Plugin</url>
+  <url>https://github.com/jenkinsci/scm-api-plugin/master/docs/consumer.adoc</url>
```

Comment:
I think I've made a mistake here.  I need to double check the instructions to confirm this is the correct way to refer to an adoc file in the GitHub repository. I suspect an error in the URL.

Reply:
you need to add tree to the URL I believe